### ISSUE TITLE
false positive: jsdoc-format

### ISSUE THREAD

**Issue by unional** [palantir/tslint#1121]
TSLint: 3.7.3

```js
  /*********
   * Chain *
   ********/
```
Not really JSDoc.... :rose:

**Comment by nomaed** [comment]
I agree.
I often use 
```
/*****************************************************************/
```
for visual clarity to separate various sections of classes (for example, a separator between properties and methods, or publics and privates).

I keep getting error although these aren't JSDocs.

I suggest that detection of a JSDoc should check for any whitespace after the `/**` sequence.
I made this change in my local build, and I will add a PR for it.

**Comment by JKillian** [comment]
Fixed by #1543
